tests/test-war-smoke/pom.xml

```diff
@@ -152,12 +121,13 @@
     </profile>
     <profile>
       <id>test.remote</id>
-      <activation>
-        <property>
-          <name>test.mode</name>
-          <value>remote</value>
-        </property>
-      </activation>
+      <properties>
+        <maven.build.timestamp.format>yyyyMMdd-HHmm</maven.build.timestamp.format>
+        <gcloud-projectId-file>${project.build.directory}/gcloud-projectid.properties
+        </gcloud-projectId-file>
+        <!-- remote tests require image in gcr.io -->
+        <jetty.test.image>gcr.io/${app.deploy.project}/jetty:${docker.tag.short}</jetty.test.image>

```

Comment:
This is problematic because the tag carries the ${maven.build.timestamp} along with it and the change in the ${maven.build.timestamp.format} between pom.xml files changes the value of the ${docker.tag.long} property.